ee/modules/021-cni-cilium/templates/egress-gateway-agent/daemonset.yaml

```diff
@@ -42,6 +42,8 @@ spec:
       maxUnavailable: 1
```

Comment:
Don't we need it here?
https://github.com/deckhouse/deckhouse/blob/main/modules/021-cni-cilium/templates/_agent-daemonset.tpl